### ISSUE TITLE
[A11Y] Clickable elements should have interactive semantics

### ISSUE THREAD

**Issue by softvision-oana-arbuzov** [webcompat/webcompat.com#3482]
**Browser / Version**: Firefox Nightly 81.0a1 (2020-08-18)
**Operating System**: Windows 10 Pro

**Steps to Reproduce**:
1. Navigate to https://webcompat.com/issues/new
2. Fill in all required info until reaching "What is wrong with the page" section.
3. Observe categories.

**Expected Behavior:**
Categories have interactive semantics.

**Actual Behavior:**
Categories don't have interactive semantics.

**Notes:**
1. Screenshot attached.
![image](https://user-images.githubusercontent.com/12184325/93611263-c31d0d00-f9d6-11ea-8887-95095efe81b0.png)

2. The issue is also reproducible for:
- browsers 
![image](https://user-images.githubusercontent.com/12184325/93611801-6a01a900-f9d7-11ea-86be-c2f4f5408076.png)

- navigation bullets from "What is a web compatibility issue?" popup
![image](https://user-images.githubusercontent.com/12184325/93611594-2d35b200-f9d7-11ea-99be-e80c332a3f7e.png)

- "X" button from "What is a web compatibility issue?" and "Public Reports" popups
![image](https://user-images.githubusercontent.com/12184325/93611653-40488200-f9d7-11ea-8f7e-26e57496af10.png)

- report via GitHub icon
![image](https://user-images.githubusercontent.com/12184325/93611469-02e3f480-f9d7-11ea-84ce-4d7bb3fa2990.png)

2. Read more about it here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard

**Watchers:**
@softvision-oana-arbuzov
@cipriansv
